addons/containerd/template/script.sh

```diff
@@ -269,6 +274,14 @@ function find_common_versions() {
             add_supported_os_to_manifest_file "$version" "ubuntu-22.04" "Dockerfile.ubuntu22"
         fi
 
+        if ! contains "$version" ${AMAZON2023_VERSIONS[*]}; then
+            echo "Amazon 2023 lacks version $version"
+            add_unsupported_os_to_preflight_file "$version" "amzn" "2023"
+        else
+            add_supported_os_to_preflight_file "$version" "amzn" "2023"
+        fi
```

Comment:
if we're not including packages we probably don't ever need to add amzn 2023 to the unsupported list, we could just always say it's supported

Reply:
This PR has no reason to exist. I will rewrite the whole thing from scratch.